readme.md

```diff
@@ -415,6 +415,7 @@
 - [Programming for Kids](https://github.com/HollyAdele/awesome-programming-for-kids#readme)
 - [Educational Games](https://github.com/yrgo/awesome-eg#readme) - Learn while playing.
 - [JavaScript Learning](https://github.com/micromata/awesome-javascript-learning#readme)
+- [CSS Learning](https://github.com/micromata/awesome-css-learning#readme) - Learning ressources regarding CSS – the language and the modules.
```

Comment:
"Learning ressources regarding CSS" is just repeating the title, also typo.

Reply:
Thanks. I’ve just changed this to:

> [CSS Learning](https://github.com/micromata/awesome-css-learning#readme) - Mainly about CSS – the language and the modules.